### ISSUE TITLE
Visible tile should be invisible

### ISSUE THREAD

**Issue by johndh** [Andrettin/Wyrmgus#76]
![screenshot from 2016-03-14 03 07 48](https://cloud.githubusercontent.com/assets/692397/13737624/0ff419cc-e995-11e5-94c0-6390caf2d684.jpg)
See the perfectly cut-out patch of forest in the midst of the shroud?  Pretty sure that shouldn't be there.

Graphics card is an AMD/ATI RS880M Mobility Radeon HD 4225/4250, in case it matters.  Running git version of Wyrmgus and Wyrmsun on 64-bit Linux Mint.

**Comment by Andrettin** [comment]
Yes, it certainly shouldn't be there, but unfortunately I haven't been able to reproduce this. Did it happen again even after the fix to the other tree tile issue?

**Comment by johndh** [comment]
I haven't experienced it recently.  It very well may be the same bug, in which case I consider it fixed until/unless I see it again.
